shiny/ui/_chat_tokenizer.py

```diff
@@ -45,12 +45,23 @@ def encode(
 TokenEncoding = Union[TiktokenEncoding, TokenizersTokenizer]
 
 
-def get_default_tokenizer() -> TokenizersTokenizer | None:
+def get_default_tokenizer() -> TokenizersTokenizer:
     try:
         from tokenizers import Tokenizer
 
         return Tokenizer.from_pretrained("bert-base-cased")  # type: ignore
-    except Exception:
-        pass
-
-    return None
+    except ImportError:
+        raise ValueError(
+            "Failed to download a default tokenizer. "
+            "A tokenizer is required to impose `token_limits` on `chat.messages()`. "
+            "To get a generic default tokenizer, install the `tokenizers` "
+            "package (`pip install tokenizers`). "
+        )
+    except Exception as e:
+        raise ValueError(
+            "Failed to download a default tokenizer. "
+            "A tokenizer is required to impose `token_limits` on `chat.messages()`. "
+            "Try downloading a different tokenizer using "
+            "`tokenizers.Tokenizer.from_pretrained()`. "
+            f"Error: {e}"
+        ) from e
```

Comment:
This change is orthogonal to the main fix of this PR, but it's a good idea, and I ran into because [bert-base-cased](https://huggingface.co/google-bert/bert-large-cased) temporarily went offline for a couple of the test runs.

Reply:
Seems like the second message is missing some instruction about how to choose a specific tokenizer that was in the original message. I also think it'd be nice to include how to turn off the need for a tokenizer, but that might be obvious from the code you'd write to get here.